docs/report-fields-description.md

```diff
@@ -0,0 +1,115 @@
+# Report Fields For Collected Metrics
+
+This document provides an outline of the fields included in the collected usage metrics. These metrics relate to containers, persistent volumes, nodes, pods, and namespaces. 
+
+
+**NOTE:**
+
+- The [Prometheus queries](https://github.com/project-koku/koku-metrics-operator/blob/main/internal/collector/queries.go) used by the operator to collect metrics are detailed in the linked file. 
+
+- To enable the collection ROS (Resource Optimization) metrics, ensure that the namespace(s) are labeled with `insights_cost_management_optimizations='true'`.
+
+    - Queries responsible for collecting ROS metrics are identified in the `QueryMap` with the prefix `ros:` and include a specific filter to target the appropriately labeled namespaces:
+        ```
+        kube_namespace_labels{label_insights_cost_management_optimizations='true', namespace!~'kube-.*|openshift|openshift-.*'}
+        ```
+
+
+### 1. Common Fields
+
+- `report_period_start`: The start timestamp of the reporting period.
+- `report_period_end`: The end timestamp of the reporting period.
+- `interval_start`: The start timestamp of the reporting interval.
+- `interval_end`: The end timestamp of the reporting interval.
+
+These common fields are included in all the reports and provide temporal information about the reporting period and interval.
+
+### 2. Container Metrics
+
+Fields for metrics related to containers:
+
+
+- `container_name`: The name of the container.
+- `pod`: The name of the pod that contains the container.
+- `owner_name`: The name of the owner entity (e.g., deployment, statefulset) associated with the container.
```

Comment:
NITPICK! :grin: And only because it looks weird against the other lines.
```suggestion
- `owner_name`: The name of the owner entity (e.g., Deployment, StatefulSet) associated with the container.
```